tests/lib-injection/dd-lib-python-init-test-django-unsupported-package-force/django_app.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+from django.http import HttpResponse
+from django.urls import path
+
+
+filepath, extension = os.path.splitext(__file__)
+ROOT_URLCONF = os.path.basename(filepath)
+DEBUG = False
+SECRET_KEY = "fdsfdasfa"
+ALLOWED_HOSTS = ["*"]
+
+
+def index(request):
+    import ddtrace
+
+    if ddtrace.__version__ != "2.6.3":
```

Comment:
Do we do this to generate an expected error path we want to test?

Reply:
There are other test cases that cover the entirety of the test matrix (with the exception of telemetry, which will be in https://github.com/DataDog/dd-trace-py/pull/9477)